kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { useContext, useRef, useState } from 'react';
+
+import { getLocale, gettext, Interpolated } from '../l10n.js';
+import UserProvider from '../user-provider.jsx';
+
+export default function SubscriptionForm() {
+    const locale = getLocale();
+    const subscriptionFormRef = useRef(null);
+    const userData = useContext(UserProvider.context);
+    const [subscribeFormDisabled, setSubscribeFormDisabled] = useState(false);
+    const [enableSubscribeButton, setEnableSubscribeButton] = useState(false);
```

Comment:
In the spirit of consistency with L11, I'd recommend going with `subscribeButtonEnabled`